installer/dmg/build-dmg.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+rm ./*.dmg || true
+rm dmg-contents/*.pkg || true
+
+VERSION=$1
+ROOT="/Library/MongoDB/AtlasSQL ODBC/$VERSION"
+mkdir -p components/"$ROOT"
+mv ./libatsql.dylib components/"$ROOT"/
+cp ./resources/*.rtf components/"$ROOT"/
+cp ../../README.md components/"$ROOT"/
+
+# build component pkg
+pkgbuild --root=components/ --scripts=scripts/ --identifier='AtlasSQL ODBC' 'mongoodbc-component.pkg'
```

Comment:
```suggestion
pkgbuild --root=components/ --scripts=scripts/ --identifier='MongoDB Atlas SQL ODBC' 'mongoodbc-component.pkg'
```
Another spot to update the name that Natacha is suggesting